app/controllers/lessons_controller.rb

```diff
@@ -4,5 +4,17 @@ def create
     @lesson=@category.lessons.create(user_id:current_user.id)
     redirect_to new_category_answer_url(@category.id)
   end
+
+  def show
+    @category=Category.find(params[:id])
+    @words=@category.words
+    @choices=Choice.where(word_id: @words.pluck(:id))
```

Comment:
@hashimoto97 Use associations in the Category model to get correct choices, like this:
`category.choices`